pkgs/applications/office/timekeeper/default.nix

```
@@ -0,0 +1,42 @@
+{ stdenv
+, fetchFromGitHub
+, poco
+, pkgconfig
+, gnome2
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "timekeeper";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "bburdette";
+    repo = "TimeKeeper";
+    rev = "363a3f0033fcca2d45b0d1f0fc74d6470c9db10d";
+    fetchSubmodules = true;
+    sha256 = "03alrfy7axq64k9lrs14afc0m6zn5gr5glhgx9i2r1yam7wdmz5c";
+  };
+
+  buildInputs = [
+    poco
+    pkgconfig
+    gnome2.gtkmm
+    gnome2.libglademm
+    gnome2.libglade
+    ];
+
+  NIX_CFLAGS_COMPILE = "-isystem ${lib.getDev gnome2.libglade}/include/libglade-2.0";
+
+  installPhase = ''
+    mkdir -p $out/bin/
```

```suggestion
    mkdir $out/bin/
```

Did you try this? Out should exist.